qiita_pet/templates/analysis_selected.html

```diff
@@ -1,8 +1,15 @@
 {% extends sitebase.html %}
 {% block head %}
 <script type="text/javascript">
+var ws = new WebSocket('ws://' + window.location.host + '/analysis/selected/socket/');
+ws.onmessage = function(evt) { if(evt.data != 'true') {ws.close();} };
```

Comment:
Well, better to understand than guess...
On Apr 18, 2015 10:30 AM, "Joshua Shorenstein" <notifications@github.com>
wrote:

> In qiita_pet/templates/analysis_selected.html
> <https://github.com/biocore/qiita/pull/1058#discussion_r28645315>:
>
> > @@ -1,8 +1,15 @@
> >  {% extends sitebase.html %}
> >  {% block head %}
> >  <script type="text/javascript">
> > +var ws = new WebSocket('ws://' + window.location.host + '/analysis/selected/socket/');
> > +ws.onmessage = function(evt) { if(evt.data != 'true') {ws.close();} };
>
> Maybe? I may be being overly cautious, but at this point my thinking is
> better safe than sorry.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/1058/files#r28645315>.
>